wiki/Tournaments/oPHN/2024_taiko/en.md

```diff
@@ -0,0 +1,395 @@
+
+### Qualifiers
+
+The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1WF-I_EkpFGLJmB-19lLCu-OpBSiXvUFj6l3QrQrsraU/edit?rm=minimal)
+
+## Ruleset
+
+### General Rules
+
+- Players must be in the rank range of #500 to #10,000 to be eligible to join.
+- Players who currently have (or had at some point since account creation) the Philippine Flag on their profile are the only ones allowed to join the tournament.
+- Score v2 will be used.
+- Permitted mods for Forced Mod (FM) and Tiebreaker picks are Hidden (HD), HardRock (HR), or both.
+- No Fail (NF) is enforced for all picks.
+
+### Round Overview
+
+#### Qualifier Stage
+
+- Players must sign up to one of the qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time.
+- There will be no warmups for all qualifiers lobbies. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1, DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.
+- The higher scores from each player for each map will be used for seeding. Players will be ordered by overall rank, which is computed by taking the average of their rankings on each map. In case of a tie, the sum of the top scores of the tied players are taken into account instead. Sharing of scores before the results showcase is discouraged.
+- The number of players that will advance to the next stage will depend on the total number of participants.
+
+#### Bracket Stage
+
+- Players are placed into a double-elimination bracket. They comprise of the Top 16 from the qualifiers.
+- Mappool Sizes and Best of Format for each round are given as follows:
+  - Round of 16 (Best of 9): 4 NM, 2 HD/HR/DT, 3 FM, 1 TB
+  - Quarterfinals and Semifinals (Best of 11): 5 NM, 2 HD/HR/DT, 3 FM, 1 TB
+  - Finals and Grand Finals (Best of 13): 6 NM, 2 HD/HR/DT, 3 FM, 1 TB
+
+### Match Rules
+
+- The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time. The multiplayer rooms are named in the following format: `PHNT2024: (<Player 1>) vs. (<Player 2>)`
+- If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
+- Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use DoubleTime if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
```

Comment:
```suggestion
- Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use Double Time if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player's warm-up is skipped.
```